src/server/examples/examples_test.go

```diff
@@ -341,3 +342,107 @@ func TestFruitStand(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("filter", false))
 	require.NoError(t, c.DeleteRepo("data", false))
 }
+
+func TestScraper(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	c := getPachClient(t)
+
+	// Check that scraper pipeline manifest appears in the docs (before we create a new repo or anything)
+	readme, err := ioutil.ReadFile("../../../doc/examples/scraper/README.md")
```

Comment:
I like that you're validating that this test stays in sync w the documentation. However, after discussing w JD I went another route to guarantee this for the other tests. Instead of having any manifest in a few places (test, readme, separate file), we opted for just a separate file. I also had to update the readme so that it told the user where to look for that file.